src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+        An entity name .
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        entityTypes
+      </td>
+
+      <td>
+        String
+      </td>
+
+      <td>
+        An entity type.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        environmentId
+      </td>
+
+      <td>
+        Int!
+      </td>
+
+      <td>
+       See [our documentation](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/) to learn about environments
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        incidentId
+      </td>
+
+      <td>
+        ID
+      </td>
+
+      <td>
+        Incident identifier.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        priority
+      </td>
+
+      <td>
+        Priority
+      </td>
+
+      <td>
+        Incident priority. Can be one of: `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`.
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        state
+      </td>
+
+      <td>
+       IncidentState.
+      </td>
+
+      <td>
+        Incident state. Can be one of: `CREATED`, `CLOSED`.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        timestamp
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+        Incident event timestamp.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        title
+      </td>
+
+      <td>
+        String
+      </td>
+
+      <td>
+        The description the user gave the condition if provided. 
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        updatedAt
+      </td>
+
+      <td>
+        EpochMilliseconds
+      </td>
+
+      <td>
+        The time the incident was last updated.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        anomalyId
+      </td>
+
+      <td>
+       Optional
+      </td>
+
+      <td>
+        If this incident originated from a New Relic anomaly it will have an anomaly ID.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        conditionFamilyId
+      </td>
+
+      <td>
+       Optional
+      </td>
+
+      <td>
+        The condition id
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        policyIds
+      </td>
+
+      <td>
+       String
+      </td>
+
+      <td>
+        The Id of the policy that generated this incident.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        aggregationTags
+      </td>
+
+      <td>
+        [KeyValue!]
+      </td>
+
+      <td>
+        Aggregation of tags related to this incident.
+      </td>
+    </tr>
+     </tbody>
+</table>
+
+
+## List and filter issues [#list-filter-issues]
+
+The `issues` query allows you to paginate through all of your issues per account. It also allows some filtering functionality on the issues.
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="listissues"
+    title="List issues"
+  >
+    If you don't provide any filtering data you will get the issue data for the last 24 hours
+
+**Request**
+
+```graphql
+{
+  currentUser {
+    account(id: ACCOUNT_ID) {
```

Comment:
note that user-specific variables in example calls (not results) should have <var> tags, so like

<var>ACCOUNT_ID</var>